routes/projectsDB.ts

```diff
@@ -1,15 +1,19 @@
 import { Router } from "express";
-import connectDB from "../connectDB";
 import express from "express";
 import logger from "../utils/logger";
 import { queryDatabase } from "./databaseFunctions";
-import { QueryResult } from "pg";
+import { Client, QueryResult } from "pg";
 import validate from "../middlewares/validate";
+import getDB from "../db";
 
 const router: Router = Router();
+const client: Client = await getDB();
```

Comment:
i don't think you can do an await outside an async call...? im  getting an error trying to run this with nodemon.

Reply:
Yea, I ran it and it worked fine. Just fix the await syntax for getting the client. It should work via the `bun serve` command I have in `package.json` AND also `bun server.ts`. After that we should be good to merge.